WcaOnRails/db/structure.sql

```diff
@@ -1048,51 +848,35 @@ CREATE TABLE `posts` (
   KEY `index_posts_on_world_readable_and_sticky_and_created_at` (`world_readable`,`sticky`,`created_at`),
   KEY `index_posts_on_world_readable_and_created_at` (`world_readable`,`created_at`),
   KEY `idx_show_wr_sticky_created_at` (`show_on_homepage`,`world_readable`,`sticky`,`created_at`)
-) ENGINE=InnoDB AUTO_INCREMENT=7114 DEFAULT CHARSET=utf8mb4;
+) ENGINE=InnoDB AUTO_INCREMENT=7780 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
-
---
--- Table structure for table `preferred_formats`
---
-
 DROP TABLE IF EXISTS `preferred_formats`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `preferred_formats` (
-  `event_id` varchar(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
-  `format_id` varchar(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+  `event_id` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `format_id` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
   `ranking` int(11) NOT NULL,
   UNIQUE KEY `index_preferred_formats_on_event_id_and_format_id` (`event_id`,`format_id`),
   KEY `fk_rails_c3e0098ed3` (`format_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
-
---
--- Temporary table structure for view `rails_persons`
---
-
 DROP TABLE IF EXISTS `rails_persons`;
 /*!50001 DROP VIEW IF EXISTS `rails_persons`*/;
 SET @saved_cs_client     = @@character_set_client;
 SET character_set_client = utf8;
-/*!50001 CREATE TABLE `rails_persons` (
-  `id` tinyint NOT NULL,
-  `wca_id` tinyint NOT NULL,
-  `subId` tinyint NOT NULL,
-  `name` tinyint NOT NULL,
-  `countryId` tinyint NOT NULL,
-  `gender` tinyint NOT NULL,
-  `year` tinyint NOT NULL,
-  `month` tinyint NOT NULL,
-  `day` tinyint NOT NULL,
-  `comments` tinyint NOT NULL
-) ENGINE=MyISAM */;
+/*!50001 CREATE VIEW `rails_persons` AS SELECT 
+ 1 AS `id`,
```

Comment:
Hmm... this seems wrong. I think we should probably remove this from the diff. Bonus points to figure out why this happened.